ONSdigital/eq-survey-runner
# Replace MCI (0215) survey with refresh

### What is the context of this PR?
Replace the MCI (0215) survey with the refreshed version created in https://github.com/ONSdigital/eq-survey-runner/pull/1138

### How to review 
- Check that choosing the `1_0215` survey launches the refreshed survey (split across multiple pages)
- Check that the `mci_refresh` survey is no longer available
